specs/latest/1.0/index.html

```diff
@@ -3481,6 +3481,14 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+    <h3>No Shader Binaries</h3>
+
+<p>
+
+Accessing binary representations of compiled shaders is not supported in the WebGL API. This includes OpenGL ES 2.0 ShaderBinary entry point. Shader compiler should be always true. In addition, querying shader binary formats and shader compiler with getparameter is not supported in the WebGL API.
```

Comment:
Shader compiler is a separate issue from shader binaries.  Let's address it in a separate point (or a separate CL).

Add <code /> around ShaderBinary and getParameter (capital P)